lib/bigtable/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
We can probably just catch all of these in the root .gitignore so each directory doesn't need a .gitignore.

I'd also like to figure out only having `AUTHORS`, `CONTRIBUTORS`, and `COPYING` in the root directory of the repo, then including them at the time of `npm publish`, but this might have to be solved later when we get to the release process tooling.